alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/gui/effects/JEffectsTab.java

```diff
@@ -263,13 +264,13 @@ public void setEffects(final List<Effect> effects) {
     @Override
     public void setEnabled(final boolean value) {
         super.setEnabled(value);
-        addEffectButton.setEnabled(value);
-        remEffectButton.setEnabled(value);
-        saveButton.setEnabled(value);
-        loadButton.setEnabled(value);
-        moveLeftButton.setEnabled(value);
-        moveRightButton.setEnabled(value);
-        for (final Component c : stackSec.getComponents()) {
+        this.addEffectButton.setEnabled(value);
+        this.remEffectButton.setEnabled(value);
+        this.saveButton.setEnabled(value);
+        this.loadButton.setEnabled(value);
+        this.moveLeftButton.setEnabled(value);
+        this.moveRightButton.setEnabled(value);
+        for (final Component c : this.stackSec.getComponents()) {
```

Comment:
Not  a problem - but why this "this" invasion? Where they necessary, or was it the setting of your editor? If so, be wary: in many cases, abusing this. leads to unnecessary hyperverbose code (and java is already verbose itselft)